pysqlexpr/table.py

```diff
@@ -241,24 +241,32 @@ def __str__(self) -> str:
 
 
 class Table:
-    __slots__ = ("name", "columns", "description")
+    __slots__ = ("name", "columns", "description", "replace")
 
     name: Identifier
     columns: list[Column]
     description: str | None
+    replace: bool
 
     def __init__(
-        self, name: str, columns: list[Column], *, description: str | None = None
+        self,
+        name: str,
+        columns: list[Column],
+        *,
+        description: str | None = None,
+        replace: bool = False
```

Comment:
Instead of adding this parameter to `__init__` (and to the class definition), I would consider creating a new function such as `as_stmt`, which would emit a SQL statement. `as_stmt` could take parameters such as `replace`, and `__str__` would invoke `as_stmt` with meaningful defaults.